src/stopwatch.js

```diff
@@ -0,0 +1,28 @@
+const namedPerformances = {}
+const defaultName = 'default'
+
+const stopwatch = () => {
+  return {
+    start: (name) => {
+      name = name || defaultName
+      namedPerformances[name] = {
```

Comment:
To be honest I feel the whole files for stopwatch is unnecessary since (1) it has only 3 useful lines and (2) is it used in just one place

Reply:
Now with the addition of the elapsed time for the step and use case I use it in two places.

Maybe you can see some refactoring to leave in the step only?

Reply:
(1): you can accomplish the same behavior using a factory method, as described
(2): since Use Case uses a Step to run, you should use it in just one place